packages/lab/src/dialog-next/DialogNext.tsx

```diff
@@ -0,0 +1,118 @@
+import {
+  forwardRef,
+  HTMLAttributes,
+  useEffect,
+  useMemo,
+  useRef,
+  useState,
+} from "react";
+import { clsx } from "clsx";
+import {
+  FloatingFocusManager,
+  FloatingOverlay,
+  FloatingPortal,
+} from "@floating-ui/react";
+import { makePrefixer, useForkRef, useId } from "@salt-ds/core";
+import { useWindow } from "@salt-ds/window";
+import { useComponentCssInjection } from "@salt-ds/styles";
+import { useDialogNext } from "./useDialogNext";
+
+import dialogNextCss from "./DialogNext.css";
+import { DialogNextContext } from "./DialogNextContext";
+import { DialogNextBody } from "./DialogNextBody";
+
+export interface DialogNextProps extends HTMLAttributes<HTMLDivElement> {
+  /**
+   * Display or hide the component.
+   */
+  open?: boolean;
+  /**
+   * Callback function triggered when open state changes.
+   */
+  onOpenChange?: (open: boolean) => void;
+  /**
+   * Status indicator
+   * */
+  status?: "info" | "success" | "warning" | "error";
```

Comment:
This should use `ValidationStatus` type like `Toast` and `Banner`.

Interestingly, the `status` prop on all 3 components has a different (and borderline useless) comment explaining what the status prop does.

Reply:
You'd be surprised how many useless comments we have 👀